__init__.py

```diff
@@ -553,8 +545,29 @@ def _get_alarm_matches(
             time_alarm = to_utc(when).timestamp()
             if is_midnight:
                 time_alarm = time_alarm + 86400.0
+
             time_matches = [a for a in alarms if abs(a["timestamp"] - time_alarm) <= 60]
 
+        # add other categories of alarm matches here
+        utt = self.workaround_lingua_franca(utt)
+        when, utt_no_datetime = extract_datetime(utt) or (None, utt)
+
+        when_utc = None
+        if when is not None:
+            when_utc = when.astimezone(pytz.utc)
```

Comment:
another possible `to_utc(when)`

Reply:
I don't really know and its taking a while to chase each of these down so I'll just change them all and rerun the tests and see if anything fails.

Reply:
I did and they passed so they have been updated.